Python/Machine Learning/Credit card approval prediction/readme.md

```diff
@@ -0,0 +1,53 @@
+# Credit card approval prediction
+
+## Description
+Commercial banks receive a lot of applications for credit cards. Many of them get rejected for many reasons, like high loan balances, low income levels, or too many 
+inquiries on an individual's credit report. This task can be automated with the power of machine learning. In this project, we will build an automatic credit card approval 
+predictor using machine learning.
+
+## Tech Stack
+Python
+
+## Package used
+ - pandas
+ - numpy
+ - matplotlib
+ - scikit-learn
+
+## Install the dependencies
+ - Install python 3.8.3 and above
+ - Further install this all libraries using terminal
+   - pip install pandas
+   - pip install numpy
+   - pip install matplotlib
+   - pip install scikit-learn
+- Clone this repo : git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+- cd Python
+- cd Machine Learning
+- cd Credit card approval prediction
```

Comment:
use code snippets for these all commands so that user can easily copy and paste it.

Reply:
Done